### PR TITLE
New package: cryfs-0.9.7

### PR DESCRIPTION
**Pull request by Anachron**
Cryptographic filesystem for the cloud

### CODE REVIEW

**Comment by Anachron** [comment]
Old build log: https://travis-ci.org/voidlinux/void-packages/builds/284237935

**Comment by Vaelatern** [comment]
Purpose of the massive patch?

File: srcpkgs/cryfs/template [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+pkgname=cryfs
+version=0.9.7
+revision=1
+create_wrksrc=yes
+build_style=cmake
+configure_args="-DCMAKE_INSTALL_PREFIX=/usr \
+	-DCMAKE_BUILD_TYPE=Release \
+	-DBUILD_TESTING=OFF \
+	-DCRYFS_UPDATE_CHECKS=OFF"
+hostmakedepends="cmake python boost"
```
Comment (Vaelatern) [inline srcpkgs/cryfs/template]: cmake redundant.

File: srcpkgs/cryfs/template [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Template file for 'cryfs'
+pkgname=cryfs
+version=0.9.7
+revision=1
+create_wrksrc=yes
+build_style=cmake
+configure_args="-DCMAKE_INSTALL_PREFIX=/usr \
+	-DCMAKE_BUILD_TYPE=Release \
```
Comment (Vaelatern) [inline srcpkgs/cryfs/template]: `cmake_args+=" -DCMAKE_INSTALL_PREFIX=/usr -DCMAKE_BUILD_TYPE=Release"`
makes these two lines redundant.

File: srcpkgs/cryfs/template [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+hostmakedepends="cmake python boost"
+makedepends="crypto++-devel fuse-devel libressl-devel boost-devel libcurl-devel"
+depends="crypto++ fuse curl"
+short_desc="Cryptographic filesystem for the cloud"
+maintainer="Anachron <Anachron14@gmx.de>"
+license="LGPL-3"
+homepage="https://www.cryfs.org"
+distfiles="https://github.com/cryfs/${pkgname}/releases/download/${version}/${pkgname}-${version}.tar.gz"
+checksum="a24f73d0162fb93c430b1866cd99814ca146d92741cc142f888c7d323a9d33c0"
+only_for_archs="i686 x86_64"
```
Comment (Vaelatern) [inline srcpkgs/cryfs/template]: please remove this line. also github UI shows no newline at end, so this is irritating.

**Comment by Anachron** [comment]
The only way to make it compile, the maintainer pushed 0.9.7 with an issue which breaks cryfs and thus prevents it from being built.

I didn't want to upload the upcoming release since its not finished yet and we have no idea what bugs are hidden inside.

The second last version 0.9.6 is quite old imo and not good to be used therefor. (especially since dependency versions changed and may not work nowadays)

**Comment by Anachron** [comment]
Related issue: https://github.com/cryfs/cryfs/issues/138

**Comment by Vaelatern** [comment]
Doesn't explain the massive diff. The patch listed at that issue is nice and small.

**Comment by Anachron** [comment]
If you mean the "fixed" one it's belonging to the ArchLinux AUR pkg, not the patch for the outdated lib itself.

This is the patch itself: https://github.com/cryfs/cryfs/commit/f1c6fa044f44e33c0c9e6eab78877d47ac4c87be

It's from smessmers comment: 
"Thanks for reporting. Seems one of the dependencies (spdlog) was using an outdated version of fmtlib. They fixed it since then, and I now updated the dependency in f1c6fa0.

Can you verify it's fixed?"

**Comment by Anachron** [comment]
Any news on this?

**Comment by Vaelatern** [comment]
https://github.com/cryfs/cryfs/issues/200#issuecomment-357021866 We will wait a few days.

**Comment by Vaelatern** [comment]
0.9.8.

**Comment by Anachron** [comment]
Thanks I'll update this PR accordingly.

**Comment by Anachron** [comment]
Is anybody still interested in this? I am not using this anymore so if there is still some interest I'll update the template, else I will close this.

**Pull request closed without merging**